runner/webserver.ts

```diff
@@ -36,6 +36,22 @@ const DEFAULT_HEADERS = {
   'Expires': '0',
 };
 
+// scripts to be injected into the running test
+const ENVIRONMENT_SCRIPTS: NPMPackage[] = [
+  {name: 'stacky', jsEntrypoints: ['browser.js']},
```

Comment:
do we have cases with more than one entrypoint?

Reply:
We do not. Do you think it's better to change` jsEntrypoints: string[]` to` jsEntrypoint: string`?

Reply:
I think it's simpler, if we're not using multiple entrypoints yet. `resolve()` works with full file paths, so you wouldn't need an object here, just strings.